web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -938,6 +938,7 @@
             createPageTpl: "../../catalog/templates/editor/new-metadata-horizontal.html",
             editorIndentType: "",
             allowRemoteRecordLink: true,
+            workflowSearchRecordTypes: ["n", "e"],
```

Comment:
It has no meaning on the backend side so it can make sense to keep it on client side only. It could be portal specific even if it probably makes more sense to have same config in all the catalogue. We also have other UI settings related to the workflow like `workflowAssistApps`. So I would keep it like this.

Reply:
ok